pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/operator/OpChainTest.java

```diff
@@ -112,7 +112,8 @@ public void setUp() {
       when(_exchange.getRemainingCapacity()).thenReturn(1);
       when(_mailbox2.poll()).then(x -> {
         if (_blockList.isEmpty()) {
-          return TransferableBlockUtils.getNoOpTransferableBlock();
+          //return TransferableBlockUtils.getNoOpTransferableBlock();
+          return TransferableBlockUtils.getEndOfStreamTransferableBlock();
```

Comment:
I guess this is the correct result right now. I need to remove the comment above